daemon/api_systems.go

```diff
@@ -578,3 +584,111 @@ func postSystemActionRemove(c *Command, systemLabel string) Response {
 
 	return AsyncResponse(nil, chg.ID())
 }
+
+type encryptionSupportInfoKey struct{ systemLabel string }
+
+func cachedEncryptionSupportInfoByLabel(c *Command, systemLabel string) (*install.EncryptionSupportInfo, error) {
```

Comment:
this could use a doc comment mentioning that the cached entry isn't cleared as there's no point in doing so